packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -532,6 +532,8 @@ function validateCfnTemplates(context: $TSContext, resourcesToBeUpdated: $TSAny[
       ignore: [parametersJson],
     });
 
+    // Turning off the error log as Geo resources are considered invalid by cfn-lint
```

Comment:
We should still lint other category cfn templates and just skip the geo template

Reply:
filtered out the geo resource templates for validation to not effect other categories.